src/samplers/cpu/linux/usage/bpf.rs

```diff
@@ -254,8 +259,8 @@ impl CpuUsage {
         let idle_prev = self.total_idle.value();
         let idle_delta =
             (self.online_cores.count as u64 * elapsed.as_nanos() as u64).saturating_sub(busy_delta);
-        self.total_idle
-            .set(elapsed.as_secs_f64(), idle_prev.wrapping_add(idle_delta));
+        let idle = idle_prev.wrapping_add(idle_delta);
```

Comment:
This is not a functional change, right?

Reply:
Correct. This just made it easier when i was doing some trace logging, so I'd rather leave it in this form. It feels a little cleaner and mirrors the busy time section more closely.